notifications.bs

```diff
@@ -782,20 +789,31 @@ method steps are:
    <li><p>If <var>permissionState</var> is {{PermissionState/"prompt"}}, then set
    <var>permissionState</var> to "<code>default</code>".
 
-   <li>
-    <p><a>Queue a global task</a> on the <a>DOM manipulation task source</a> given <var>global</var>
-    to run these steps:
+   <li><a>Resolve a permission request</a> with <var>promise</var>, <var>permissionState</var>, and
+   <var>deprecatedCallback</var>, if given.
+  </ol>
 
-    <ol>
-     <li><p>If <var>deprecatedCallback</var> is given, then <a for=/>invoke</a>
-     <var>deprecatedCallback</var> with « <var>permissionState</var> ». If this throws an
-     exception, then <a>report the exception</a>.
+  <li><p>Return <var>promise</var>.
+</ol>
 
-     <li><p><a for=/>Resolve</a> <var>promise</var> with <var>permissionState</var>.
-    </ol>
-  </ol>
+<p>To <dfn>resolve a permission request</dfn> given promise <var>promise</var>,
+[=permission state=] <var>permissionState</var>, and optional callback function
```

Comment:
My first PR had the type for permissionState as [`{{PermissionStatus/state}}`](https://w3c.github.io/permissions/#state-attribute), but I think that was a bug. Here it's a [`[=permission state=]`](https://w3c.github.io/permissions/#dfn-permission-state), that way we can pass in "denied" explicitly for the non-secure branch before we even have a PermissionStatus instance. (And `{{PermissionStatus/state}}` is ultimately set to `[=permission state=]` after the permission query algorithm runs).

Reply:
I'm not sure this works since permission state belongs to something else. Perhaps this should just be a string?

Reply:
Yeah, string makes more sense. Will do.